src/main/java/seedu/address/logic/commands/MarkPaidCommand.java

```diff
@@ -19,19 +20,22 @@ public class MarkPaidCommand extends Command {
     public static final String COMMAND_WORD = "markpaid";
 
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Marks that the student has paid fees for the month"
-            + "Parameters: INDEX (must be a positive integer) "
-            + "Example: " + COMMAND_WORD + " 1";
-    public static final String MESSAGE_MARKED_PAID_SUCCESS = "Mark fees paid for Person %1$s";
-    public static final String MESSAGE_ARGUMENTS = "Index: %1$d";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Marks that the amounts of fees the student has paid "
+            + "Parameters: \n"
+            + "INDEX (must be a positive integer) \n"
+            + "AMOUNT (must be a positive integer) \n"
+            + "Example: " + COMMAND_WORD + " 1" + "200";
+    public static final String MESSAGE_MARKED_PAID_SUCCESS = "Fees updated for Person %1$s";
```

Comment:
Hmmm I was thinking instead of using Person %1$s, would it be better if we use the name of the student instead for clarity? Jus a thought.

Reply:
I think that's a good idea too